pants.ini

```diff
@@ -2,7 +2,7 @@
 pants_version: 1.16.0.dev3
 
 plugins: [
-    'pantsbuild.pants.contrib.go==%(pants_version)s',
```

Comment:
Do we not need `go` anymore? If so, maybe worth removing in a separate PR. This is a nit, feel free to ignore it :)

Reply:
No, from what I can tell we were solely using `go` as a sanity check that any contrib package worked in general. From what I can tell, there was no particular reason to choose `go` and `mypy` will get the same job done, while allowing us to also type check our code.